types/sequelize/index.d.ts

```diff
@@ -3759,22 +3768,22 @@ declare namespace sequelize {
          *
          * @see    {Sequelize#query}
          */
-        findAll(options?: FindOptions): Promise<TInstance[]>;
```

Comment:
@andythecoderman unfortunately not. These lines were the whole idea of the change. I wanted to make `findAll` more strict, so it would catch things like misspelled a column names. Which means we have to use the generic `FindOptions<T>` here. The issue with type aliases sounds like a compiler limitation more than an issue with these type definitions though. Have you tried using an interface instead of a type alias? e.g.
```TypeScript
interface IYourModel extends IModel1, IModel2 {}
```